BlueConduit/copper
# Add loading states and empty states

This PR adds a few improvements Copper and the Data Table element. 

- Adds a named slot to `<DataTable/>` for an empty state. Someone can provide their own empty state for the table, or use its fallback content. 
- Adds styles and stories for a loading state, which we're calling "sketches"

### Testing and reviewing

Since most other people have probably not started using `pnpm` yet:
- Get `pnpm` installed, [see here](https://pnpm.io/installation#using-npm). Ping me if you have questions about it! 
- From the root directory run `pnpm install` followed by `pnpm build`

Now you should be good to run storybook again:
- Move to the `packages/copper-vue` directory and run `pnpm storybook` 
- View the "Async Data Table" story to see an example of the new data table loading state
- View the "Empty Data Table" story to see examples of a data table with no rows to display
